# if bag param is true in gazebo_run, will dump bags to this location
bag_dir: "/home/bezzo/catkin_ws/src/mpcc/bags"
log_dir: "/home/bezzo/catkin_ws/src/mpcc/bags/results/sac_logs"
db_file: "/home/bezzo/catkin_ws/src/mpcc/bags/results/rl_learning.db"
model_file: "/home/bezzo/catkin_ws/src/mpcc/bags/results/sac_policy.pth"

# training hyper params
batch_size: 512
num_updates: 500
latent_dims: 5
hidden_dims: 256

state_dim: 13
action_dim: 2

buffer_size: 50000

